giftless/storage/google_cloud.py

```diff
@@ -97,21 +103,84 @@ def _get_blob_path(self, prefix: str, oid: str) -> str:
             storage_prefix = self.path_prefix
         return os.path.join(storage_prefix, prefix, oid)
 
-    def _get_signed_url(self, prefix: str, oid: str, expires_in: int, filename: Optional[str] = None) -> str:
-        bucket = self.storage_client.get_bucket(self.bucket_name)
-        blob = bucket.blob(self._get_blob_path(prefix, oid))
-        token_expires = (datetime.now(tz=timezone.utc) +
-                         timedelta(seconds=expires_in))
-
-        extra_args = {}
-        if filename:
-            extra_args['content_disposition'] = f'attachment; filename="{filename}"'
-
-        url = blob.generate_signed_url(
-            response_disposition=extra_args,
-            access_token=self.api_key,
-            expiration=token_expires, version='v4')
-        return url  # type: ignore
+    def _get_signed_url(self, object_name: str,
+                        subresource=None, expiration=604800, http_method='GET',
+                        query_parameters=None, headers=None) -> str:
+        log.info("OBJ NAME")
```

Comment:
1. This should probably be a debug level message

2. You should create a specific log object for this module (e.g. do `log = logging.getLogger(__name__)` at the top of the module or in `__init__`), this allows filtering logs and setting different levels to different loggers (e.g. I can run with all logs in `warn` level but enable debug logging just for the gcp backend).

2. Please try to keep logs to one line at a time, and make them more clear. I do not know what is "OBJ NAME" (and why allcaps?). I like to think about logging (esp. debug level logging) as an alternative to comments: for someone reading the code, they can look debug log lines to get better understanding of what complex code does (or to phrase it in a reverse way, when I feel the need to put a comment explaining my code, if I can't make the code more obvious, I put a debug level log line instead). So I would prefer it if this line was something like:

```python
log.debug("Generating signed URL for object name: %s", object_name)
```

BTW, the use of `%s`, while not common in modern Python, is highly encouraged when logging, because it means `logging` will not even compose the string if `debug` level logging is not enabled.

Reply:
Removing logs; sorry, I forgot them in the code